open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/store/OMAGServerConfigStore.java

```diff
@@ -18,6 +20,17 @@
     <T> void saveServerConfig(T configuration);
 
 
+    /**
+     * Retrieve the configuration saved from a previous run of the OMAG Server Platform.
+     * (Note: this exists primarily for backwards compatibility -- wherever possible, use the parameterized version
+     * of this method.)
+     *
+     * @return server configuration
+     * @see #retrieveServerConfig(Class)
+     */
+    OMAGServerConfig retrieveServerConfig();
```

Comment:
Maybe add @Deprecated on the method

Reply:
and eventually use default implementation ( java 8 ), so no need to be implemented in our own implementations

`
default OMAGServerConfig retrieveServerConfig(){
        return retrieveServerConfig(OMAGServerConfig.class);
};
`